README.md

```diff
@@ -55,6 +55,26 @@ This is the place to:
 Note: there is no need to repeat everything that is already well-documented in
 the code.
 
+If you would like to include math equations to explain parts of your code or repo,
+it is recommended to use an external renderer and insert prepared svg files here.
+For example, it's possible to insert a link to the online CodeCogs equation editor;
```

Comment:
The quoted links will show up on the rendered README looking like a broken links at first glance. 
Why not something like this, 

"You can include explanatory math equations in either of the following two ways:

* Link to the online [CodeCogs](http://latex.codecogs.com/) equation editor as shown below
  <img src="https://latex.codecogs.com/svg.latex?\large%20x_{ik}+x_{jk}-2x_{ik}x_{jk}">
* Link to an image in the in the `readme_imgs` folder as shown below
   ![Expression](readme_imgs/expression.svg)

Users will look at the raw REAME file to see the formats.

Reply:
That's a fair suggestion - I have slight preference for being overly explicit with what we are recommending so that it's painfully obvious to users how to reproduce what's on the readme. It doesn't seem to me that the quoted links would look broken given the text around them, but I will defer on this.